<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube Multi-Downloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="url_label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>URL :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="url_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Place the video playlist url here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="location_label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save Location : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose where do you want to save the playlist&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="quality_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quality : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="quality_comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pick the quality you want&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>1080p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>720p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>480p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>360p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>240p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>144p</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;the Prograss of downloading the videos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="current_video">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Video : </string>
    </property>
   </widget>
   <widget class="QLabel" name="total_video">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Total Videos : </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fetch_urls">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;get the urls from the playlist&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Fetch URLs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_download">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;start downloading the playlist of the videos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start Downlaod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_downloading">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stop the process of downloading&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Stop Downloading</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="status_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>481</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
